typing/mode.ml

```diff
@@ -2349,10 +2369,11 @@ module Value_with (Areality : Areality) = struct
 
     let all =
```

Comment:
What is the intended order here? Should it match the order in `compare`?

Reply:
I think it currently doesn't have to, but yes it doesn't hurt to have it ordered.